src/Database/PostUpdate.php

```diff
@@ -375,4 +376,49 @@ private static function update1281()
 
 		return false;
 	}
+
+	// Post-update script of PR 5596
+	function fixGenderStrings() {
```

Comment:
The post-update function naming scheme is `updateXXXX` where XXXX is the new database version number that should trigger the post update function. It doesn't matter if there aren't any actual database change, you can just bump the database version number in `config/dbstructure.config.php`.

Reply:
We do have two different ways of processing updates after the update. There is the update.php and the postupdate.php. The latter one is only meant for stuff that will last some more time and that can be processed while the system is running.

This one more calls for a place in the update.php.